acme/acme/challenges_test.py

```diff
@@ -264,7 +264,9 @@ def test_gen_verify_cert_gen_key(self):
 
     def test_verify_bad_cert(self):
         self.assertFalse(self.response.verify_cert(
-            test_util.load_cert('cert.pem')))
+            OpenSSL.crypto.load_certificate(
```

Comment:
Because `verify_cert` calls `crypto_util._pyopenssl_cert_or_req_san` which doesn't work with a `ComparableX509` object anymore.

I'm happy to change `_pyopenssl_cert_or_req_san` to have it work with both if you'd like. I'd just have to special case that type.